ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/[externalID]/ResyncModal.tsx

```diff
@@ -14,9 +19,16 @@ type Props = {
 };
 
 export default function ResyncModal({ isOpen, onClose, url }: Props) {
+  const [overrideValue, setOverrideValue] = useState(url);
+  const [isURLOverridden, setURLOverridden] = useState(false);
   const [failure, setFailure] = useState<RegistrationFailure>();
   const [isSyncing, setIsSyncing] = useState(false);
 
+  let input = url;
+  if (isURLOverridden) {
+    input = overrideValue;
+  }
```

Comment:
```suggestion
  if (isURLOverridden) {
    url = overrideValue;
  }
```

Actually, we should override the `url` variable when overriding is enabled. That fixes an issue where we're using `url` in the `onSync` function and reads a little more clearly